# REQUEST

# name of specific object that should be grasped in the modelled scene 
string object_id

# this defines a prioritized list of object aliases that may refer to alternative grasp databases
# that have been generated on similar object types 
string[] object_types

# pose of object same object
geometry_msgs/Pose object_pose

# An optional representation of the sensed world   
sensor_msgs/PointCloud2[] cloud_external

# the name of the frame in which the sensor data is represented
string sensor_frame_name

# The maximum number of grasps to find
int32 max_nr_grasps

# The maximum time in ms to spend finding grasps if <=0 then it will take as much time as needed
int32 max_time

---

# RESPONSE

# the targets found within the specified time 
GraspTarget[] targets

# an optional quality score for each target
float64[] target_quality